package.json

```diff
@@ -59,7 +59,6 @@
   },
   "dependencies": {
     "axios": "^1.1.3",
-    "electron-packager": "^15.5.1",
```

Comment:
I think doing it this way would hose anyone NOT using docker as they would still need electron-packager. Can you explain your thought process on this one?